### PR TITLE
chore: add missing metadata to the toml files

### PR DESCRIPTION
**Pull request by zvolin**
This requires #161 for the readmes. See the last commit for diff as for now

### CODE REVIEW

File: cli/Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,19 @@ name = "lumina-cli"
 version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
+description = "Celestia data availability node implementation in Rust"
+repository = "https://github.com/eigerco/lumina"
+# readme = "README.md" -- It's using the project's root readme as for now, should it have a separate one?
```
Comment (zvolin) [inline cli/Cargo.toml:8]: how should we handle this?

Comment (fl0rek) [reply cli/Cargo.toml:8]: I guess root `README` could link to a separate `README` in `cli`, but I don't think it's worth it right now. So I'd vote for leaving it as is.

**Review by fl0rek — APPROVED** [review]
closer and closer to the release ✨

File: cli/Cargo.toml (hunk 1 above)
Comment (zvolin) [reply cli/Cargo.toml:8]: there is also a symlink way

File: blockstore/Cargo.toml [hunk 2]
```diff
@@ -3,6 +3,13 @@ name = "blockstore"
 version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
+description = "An IPLD blockstore capable of holding arbitrary data indexed by CID"
+repository = "https://github.com/eigerco/lumina"
+readme = "README.md"
+# crates.io is limited to 5 keywords and 5 categories
+keywords = ["blockstore", "ipld", "cid"]
```
Comment (oblique) [inline blockstore/Cargo.toml:10]: add `bitswap` too

**Review by oblique — APPROVED** [review]

**Pull request merged**